source/kubernetes/create-app/index.html.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Create a new application on the GOV.UK Kubernetes Platform
+weight: 20
+layout: multipage_layout
+---
+
+# Create a new application on the GOV.UK Kubernetes Platform
+
+## Assumptions
+
+This guide assumes the following:
+
+- You have completed all the steps in the ["get started" guide](/manual/get-started.html) of the
+  developer docs
+- You are deploying an application named `example` (make sure to replace this with your
+  application name in all steps as needed)
+- You have generated an initial application and pushed it to a Github repository under the
+  `alphagov` organisation with the `govuk` and `container` labels applied (this triggers setup of a
+  number of Github Actions secrets from the organisation and permissions required for workflows)
+- You have set up the application in [GOV.UK Docker](https://github.com/alphagov/govuk-docker) (not
+  covered by this guide) so that others can run it locally (this will also help you understand what
+  environment configuration your application needs, especially if it is an existing application
+  rather than a brand new one)
```

Comment:
Don't think this assumption necessary to make to get an application deployed to our k8s cluster. Maybe remove it here? or make it a suggestion later on?

Reply:
Maybe what's more pertinent is that the application is containerised? I.e. has a Dockerfile